CHANGELOG.md

```diff
@@ -11,6 +11,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `items` schema for `.controlPlane.availabilityZones` in `values.schema.json`.
 
+## [0.17.1] - 2023-02-10
```

Comment:
please add this under `unrelased` section. that is updated with versions by ci jobs.

Reply:
Moved it under unreleased section.